web/styles/subscriptions.css

```diff
@@ -241,6 +241,16 @@ h4.user_group_setting_subsection_title {
     }
 }
 
+.subscriber-search {
+    @media (max-width: $ms_min) {
+        float: none;
+        display: flex;
+        justify-content: flex-start;
+        align-items: center;
+        margin: 0 0 5px;
+    }
+}
```

Comment:
Does the same bug happen in the group settings panel? If so, you should put this inside the previous block rather than making a new one.

Reply:
It does! I hadn't realised that panel could also be affected, thanks for the catch.

Here's a before and after for the change in response to this comment:
Before | After
--- | ---
<img width="506" alt="Screenshot 2024-03-26 at 10 14 52 AM" src="https://github.com/zulip/zulip/assets/72880238/80aab863-617b-4591-ade8-ae5848e90c67"> | <img width="506" alt="Screenshot 2024-03-26 at 10 16 18 AM" src="https://github.com/zulip/zulip/assets/72880238/d568789e-ab95-4b99-8063-49e8dd12bcf4">